input/taproot_test.go

```diff
@@ -207,13 +212,10 @@ func htlcSenderTimeoutWitnessGen(sigHash txscript.SigHashType,
 	}
 }
 
-// TestTaprootSenderHtlcSpend tests that all the positive and negative paths
-// for the sender HTLC tapscript tree work as expected.
```

Comment:
~~Was the removal of the comment intentional?  Occurs a few times~~
ah i see it's moved